sdk/ml/azure-ai-ml/azure/ai/ml/entities/_deployment/data_collector.py

```diff
@@ -51,7 +51,8 @@ def _to_dict(self) -> Dict:
 
     @classmethod
     def _from_rest_object(cls, rest_obj: RestDataCollector) -> "DataCollector":
-        collections = {}
+        collections = dict()
```

Comment:
`{}` is a `dict` already, why the signature change?

Reply:
By signature change you mean the "commit delta", the function signature does not change though. I just wanted to be explicit.